rsearch/bin/main.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	search "github.com/romana/contrib/rsearch"
+	"log"
+	//	"net/http"
+	"encoding/json"
+
+	//	"io"
+)
+
+func main() {
```

Comment:
There are no comments in or for this function. Would be nice if I wouldn't have to read all the code to have an idea of what's going on.